src/components/GasStation/Slider/index.tsx

```diff
@@ -4,7 +4,7 @@ import css from './styles.module.css'
 
 const SLIDER_ITEMS = 7
 
-const Slider = ({ text }: BaseBlock) => (
+const Slider = ({ text }: Partial<BaseBlock>) => (
```

Comment:
Using `Partial` means every property may be `undefined`.

```suggestion
const Slider = ({ text }: Pick<BaseBlock, 'text'>) => (
```